app.js

```diff
@@ -130,10 +133,10 @@ messageAllMyStreaks = msg => {
   console.log(`${msg.author.username} requested their streaks via DM`)
   const streaks = db.getUserActiveStreaks(msg.author.id)
   for(let streak of streaks) {
-    const hasStreakedToday = db.hasStreakedToday(msg.author.id, streak.channelName)
+    const hasStreakedToday = db.hasStreakedToday(msg.guild.id, msg.author.id, streak.channelName)
     let postedString = 'but you haven\'t increased your streak yet today 😟'
     if(hasStreakedToday) postedString = 'and you\'ve increased your streak today 👍'
-    msg.reply(`Your ${streak.channelName} streak is currently ${streak.streakLevel} ${streak.streakLevel === 1 ? 'day' : 'days'} ${postedString}`)
+    msg.reply(`Your ${streak.guildID}-${streak.channelName} streak is currently ${streak.streakLevel} ${streak.streakLevel === 1 ? 'day' : 'days'} ${postedString}`)
```

Comment:
I think it's fine to just use the channel name, I don't know if it's necessary to show `312321312-gameplay`